src/components/ui/command.tsx

```
@@ -0,0 +1,175 @@
+      data-slot="command"
+      className={cn(
+        "bg-popover text-popover-foreground flex h-full w-full flex-col overflow-hidden rounded-md",
+        className
+      )}
+      {...props}
+    />
+  )
+}
+
+function CommandDialog({
+  title = "Command Palette",
```

ESLint

[error] Replace `"Command·Palette"` with `'Command·Palette'`

(prettier/prettier)